Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -510,6 +553,18 @@ AppDelegateViewModelOutputs {
 
     // Koala
 
+    self.notificationAuthorizationStatusProperty.signal
+      .skipNil()
+      .skip(until: self.notificationAuthorizationCompletedProperty.signal)
+      .take(first: 1)
+      .observeValues {
+        if $0.isAuthorized {
+          AppEnvironment.current.koala.trackPushPermissionOptIn()
+        } else if $0.isDenied {
+          AppEnvironment.current.koala.trackPushPermissionOptOut()
```

Comment:
Gina, this is related to your comment above and I agree it is much nice with the exhaustive enum but could't couldn't figure out how to do it.